staking-farm/src/internal.rs

```diff
@@ -131,6 +135,10 @@ impl StakingContract {
     }
 
     pub(crate) fn inner_unstake(&mut self, account_id: &AccountId, amount: u128) {
+        if self.paused {
```

Comment:
this dangerous - because it effectively means that owner can block people from unstaking from their pool.